csrc/scheduler/pointwise.cpp

```diff
@@ -700,41 +707,58 @@ void schedulePointwise(Fusion* fusion, const PointwiseParams* pparams) {
       reference_tv->split(1, NamedScalar::getParallelDim(ParallelType::TIDx));
       // [outer | i-remainder, TIDx, Vect]
 
-      reference_tv->split(0, pparams->unroll_factor);
-      // [o-remainder, Unroll| i-remainder, TIDx, Vect]
+      if (pparams->unroll_factor_inner > 1) {
+        reference_tv->split(1, pparams->unroll_factor_inner);
```

Comment:
we are splitting on dimension 1? which is the TIDx here right?

Reply:
This is for 2D scheduler, start with `[outer dim, inner dim]`, so here dimension  1 is `i-remainder` in `[0-outer | 1-i-remainder, 2-TIDx, 3-Vect]`. `i-remainder` means `what is left after splitting out other dims, e.g. Vect, TIDx`

Reply:
So this is a behavior change then.

If we look at the above commented code change, we are doing
```
-      reference_tv->split(0, pparams->unroll_factor);
-      // [o-remainder, Unroll| i-remainder, TIDx, Vect]
+      if (pparams->unroll_factor_inner > 1) {
+       reference_tv->split(1, pparams->unroll_factor_inner);
```

Which means the old behavior (outer unroll) is being updated to a default inner unroll instead?

Reply:
Good point. Should assign unroll to inner dim only when the scheduler is 1D, for 2D should assign to outer dim. 
```
    // for 1D scheduler, unroll the inner dimension
    // since there is no outer dimension.
    if (break_point == 0) {
      params->unroll_factor_inner = total_unroll;
      params->unroll_factor_outer = 1L;
    } else {
      // for 2D scheduler, unroll the outer dimension
      // to prioritize resue across different rows, will
      // be revised in heuristics tuning, e.g. unroll different
      // dims based on the broadcast dimension.
      params->unroll_factor_inner = 1L;
      params->unroll_factor_outer = total_unroll;
    }
```